src/include/migraphx/op/argmin.hpp

```diff
@@ -21,14 +22,16 @@ struct argmin
         return pack(f(self.axis, "axis"));
     }
 
+    value attributes() const { return {{"axis", axis}}; }
```

Comment:
I dont think we need a different attribute name for a single axis. We can tell whether its axes or axis by checking the type(ie `val.is_array()`).